homeassistant/util/yaml.py

```diff
@@ -31,11 +31,17 @@ class NodeStrClass(str):
         """Wrapper class to be able to add attributes on a string."""
 
         pass
+    
+    class NodeDictClass(dict):
+        """Wrapper class to be able to add attributes on a dict."""
 
+        pass
     if isinstance(obj, list):
         obj = NodeListClass(obj)
     if isinstance(obj, str):
         obj = NodeStrClass(obj)
+    if isinstance(obj, dict):
```

Comment:
(but not sure where this comes from, since even load_yaml should return OrderedDict today)